pom.xml

```diff
@@ -3,25 +3,30 @@
     <parent>
         <groupId>org.jenkins-ci.plugins</groupId>
         <artifactId>plugin</artifactId>
-        <version>1.609.1</version>
+        <version>2.14</version>
+        <relativePath/>
     </parent>
     <artifactId>mock-slave</artifactId>
     <version>1.10-SNAPSHOT</version>
     <packaging>hpi</packaging>
-    <name>Mock Slave Plugin</name>
+    <name>Mock Agent Plugin</name>
     <url>https://wiki.jenkins-ci.org/display/JENKINS/Mock+Slave+Plugin</url>
     <licenses>
         <license>
             <name>MIT License</name>
-            <url>http://www.opensource.org/licenses/mit-license.php</url>
+            <url>https://opensource.org/licenses/MIT</url>
         </license>
     </licenses>
     <scm>
-        <connection>scm:git:git://github.com/jenkinsci/mock-slave-plugin.git</connection>
-        <developerConnection>scm:git:git@github.com:jenkinsci/mock-slave-plugin.git</developerConnection>
-        <url>https://github.com/jenkinsci/mock-slave-plugin</url>
-      <tag>HEAD</tag>
-  </scm>
+        <connection>scm:git:git://github.com/jenkinsci/${project.artifactId}-plugin.git</connection>
+        <developerConnection>scm:git:git@github.com:jenkinsci/${project.artifactId}-plugin.git</developerConnection>
+        <url>https://github.com/jenkinsci/${project.artifactId}-plugin</url>
+        <tag>HEAD</tag>
+    </scm>
+    <properties>
+        <jenkins.version>1.609.3</jenkins.version>
```

Comment:
Oh? AFAIK it [worked fine for Pipeline](https://github.com/jenkinsci/pipeline-plugin/blob/1.14.x/pom.xml#L52).